Dockerfile

```diff
@@ -0,0 +1,11 @@
+FROM node:lts-alpine
+
+RUN npm install -g http-server
+WORKDIR /app
+COPY package*.json ./
+RUN apk add python3 make g++
```

Comment:
Add those dependencies is really useful

Reply:
Yeah, it won't compile if those aren't there.
Testing if we can remove them after running the node install string.

Reply:
Addressed this problem in https://github.com/GeoGuess/GeoGuess/pull/436/commits/a4c332272b6737924ae81b7e9278d9c7b70c42dc @BilelJegham

Reply:
You right Pyhon & g++ need for node-gyp.